bn_deprecated.c

```diff
@@ -113,10 +113,9 @@ mp_err mp_toom_sqr(const mp_int *a, mp_int *b)
 #ifdef S_MP_REVERSE_C
 void bn_reverse(unsigned char *s, int len)
 {
-   if (len < 0) {
-      return MP_VAL;
```

Comment:
this was an error. I don't know how it could manage to get into develop?

Reply:
See here https://github.com/libtom/libtommath/blob/c5b901b917c613a8b21064f816e6a7c70ed2fa09/bn_deprecated.c#L117

Reply:
lol nice

I wonder how no compiler has complained about that!

Reply:
My compiler does. Maybe the CI was not running for some reason on develop.